cmd/convert.go

```diff
@@ -41,21 +41,17 @@ Example:
 `
 
 type convertOptions struct {
-	project              string
-	ancestry             string
-	offline              bool
-	errorLogger          *zap.Logger
-	outputLogger         *zap.Logger
-	useStructuredLogging bool
-	readPlannedAssets    tfgcv.ReadPlannedAssetsFunc
+	project           string
+	ancestry          string
+	offline           bool
+	rootOptions       *rootOptions
```

Comment:
Given any thought to https://golang.org/doc/effective_go#embedding? Seems like it fits what you're doing here pretty well.

Reply:
I did consider it, but I opted not to. Basically my thought process was that the child options are _using_ rootOptions rather than _extending_ it, if that makes sense? LMK if you have strong feelings either way; I'm happy to change it.

Reply:
No strong feelings!

Reply:
Okay, then I will merge as-is.